lib/phoenix/test/channel_test.ex

```diff
@@ -256,22 +256,29 @@ defmodule Phoenix.ChannelTest do
   the result of the handler's `connect/2` callback.
   """
   defmacro connect(handler, params) do
+    quote do
+      connect(unquote(handler), unquote(params), %{})
```

Comment:
We should not have a macro that calls another macro.  In this case `connect/2` and `connect/3` should have a private function they call that emits the quoted expression. :)